### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by chandrarenu**
Hi team, I have added a new route, please review this work.

### CODE REVIEW

**Review by adarshdoesntcode — CHANGES_REQUESTED** [review]
route function in the root path

**Review by arinodes — APPROVED** [review]

**Comment by arinodes** [comment]
approved

File: main.py [hunk 1]
```diff
@@ -10,4 +10,9 @@ def get_task():
 @app.route("/test", methods = ["GET"])
 def test_Function():
   return "Test runs successful"
-app.run(debug=True)
\ No newline at end of file
+app.run(debug=True)
+
+@app.route("/", methods = ["GET"])
```
Comment (adarshdoesntcode) [inline main.py:15]: route function in the root path

File: main.py [hunk 2]
```diff
@@ -10,4 +10,9 @@ def get_task():
 @app.route("/test", methods = ["GET"])
 def test_Function():
   return "Test runs successful"
-app.run(debug=True)
\ No newline at end of file
+app.run(debug=True)
```
Comment (nabinkhadka) [inline main.py:13]: Move this line to bottom.

**Review by arinodes — APPROVED** [review]

**Review by adarshdoesntcode — APPROVED** [review]

**Pull request merged**